src/components/UserInputFormArea.tsx

```diff
@@ -7,66 +7,26 @@ export type StationInfo = {
     latitude: number;
     longitude: number;
     aliases: string[];
-    codes: {
-        id: number;
-        crs: string;
-        nlc: string;
-    };
+    id: number;
+    crs: string;
+    nlc: string;
+    isGrownStation : boolean; 
+    isSilverSeekStation : boolean;
```

Comment:
Are you ever going to use line 13&14?

Reply:
we used it to match the request response, and it might be useful in the future